src/EPR.PRN.Backend.Data/Migrations/20240730132550_InitialRecyclingTargets.cs

```diff
@@ -33,7 +33,7 @@ protected override void Up(MigrationBuilder migrationBuilder)
 
             migrationBuilder.InsertData(
                 table: "RecyclingTargets",
-                columns: new[] { "Year", "AluminiumTarget", "GlassRemeltTarget", "GlassTarget", "PaperTarget", "PlasticTarget", "SteelTarget", "WoodTarget" },
+                columns: ["Year", "AluminiumTarget", "GlassRemeltTarget", "GlassTarget", "PaperTarget", "PlasticTarget", "SteelTarget", "WoodTarget"],
```

Comment:
here too

Reply:
Yes, that's suggestion from Visual Studio / GitHub. Hence, changed